CHANGELOG.md

```diff
@@ -26,6 +70,7 @@
   * [Protocol] Improve `Protocol.UndefinedError` messages to also include the type that was attempted to dispatch on
   * [Protocol] Optimize performance of dynamic dispatching for non-consolidated protocols
   * [Record] Include field names in generated type for records
+  * [Regex] Automatically recompile regexes
   * [Registry] Add `Registry.select/2`
   * [System] Add `System.restart/0`, `System.pid/0` and `System.no_halt/1`
   * [System] Add `System.get_env/2`, `System.fetch_env/1`, and `System.fetch_env!/1`
```

Comment:
A few lines below:

> [ExUnit.DocTest] Display the actual doctest code when ~doctset~ doctest fails

In addition, I would add backticks to `--remsh`, for example:

> [IEx.CLI] Copy ticktime from remote node on IEx `--remsh`